src/lib_json/json_reader.cpp

```diff
@@ -18,6 +18,8 @@
 #include <memory>
 #include <set>
 #include <limits>
+#include <stdio.h>
```

Comment:
I'm not sure we can continue to support VS2010 for the `master` branch. `0.y.z` will work fine.

But could you try `#include <cstdio>` and `#include <cstring>` in VS2010, and if it works, push that to this branch to update this pull-request?